platform/ext/common/gcc/tfm_common_s.ld

```diff
@@ -589,6 +598,32 @@ SECTIONS
 
 #endif /* TFM_LVL != 1 */
 
+#ifndef TFM_MULTI_CORE_TOPOLOGY
+    /*
+     * Place the CMSE Veneers (containing the SG instruction) after the code, in a
+     * separate 32 bytes aligned region so that the SAU can programmed to just set
+     * this region as Non-Secure Callable.
+     */
+    .gnu.sgstubs : ALIGN(32)
+    {
+        SG_ALIGN;
+        Load$$LR$$LR_VENEER$$Base = ABSOLUTE(.);
+        *(.gnu.sgstubs*)
+    } > VENEERS AT> VENEERS
```

Comment:
For `CMSE_VENEER_REGION_IN_CODE`, there is no output region `VENEERS`, so the above looks a bit weird (it should be `> FLASH` in that case).

Reply:
I wanted to avoid `ifdef`ing this. Should I instead change it to VENEERS_REGION and define that to either FLASH or VENEERS?

Reply:
Well, perhaps `#ifdef`ing here would be cleaner, however. You wouldn't need to define `SG_ALIGN` and `SG_END_ALIGN` then. The comment before the `.gnu.sgstubs` section is also not perfectly accurate for the `CMSE_VENEER_REGION_IN_CODE` code.

Reply:
I can ifdef just that line if you really prefer that, but I really don't want to duplicate the linker code since it's very finicky, and I want to make sure the implementations don't go out of sync with each other.

I'll also fix the comment above.

Reply:
Take a look now.